src/snowflake/cli/api/secure_utils.py

```diff
@@ -14,9 +14,50 @@
 
 import stat
 from pathlib import Path
+from typing import List
 
+from snowflake.connector.compat import IS_WINDOWS
 
-def file_permissions_are_strict(file_path: Path) -> bool:
+
+def _get_windows_whitelisted_users():
+    # whitelisted users list obtained in consultation with prodsec: CASEC-9627
+    import os
+
+    return ["SYSTEM", "Administrators", os.getlogin()]
+
+
+def _run_icacls(file_path: Path) -> str:
+    import subprocess
+
+    return subprocess.check_output(["icacls", str(file_path)], text=True)
```

Comment:
No way to run this from python API?

Reply:
No, one lib had license issues, the other one lighted red light in prodsec quarters after I installed it for testing